From ad30aecd3910eef5e0849ab567c52288c515b201 Mon Sep 17 00:00:00 2001
From: Alexandru Ardelean <ardeleanalex@gmail.com>
Date: Wed, 27 Feb 2019 12:18:17 +0200
Subject: [PATCH] python3-pip: fix install rule

This seems to have slipped for some time. No idea if it ever worked.
It could be that this worked at some point.

In any case, the shebang is properly updated now.

Signed-off-by: Alexandru Ardelean <ardeleanalex@gmail.com>
(cherry-picked from commit 1b96dc01715f4f03f758ff1d087caf15726016af)
Signed-off-by: Josef Schlehofer <pepe.schlehofer@gmail.com>
(bump PKG_RELEASE for python3-pip)
---
 lang/python/python3-version.mk                   | 2 +-
 lang/python/python3/files/python3-package-pip.mk | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/lang/python/python3-version.mk b/lang/python/python3-version.mk
index 953e4b3ed..badfa1c49 100644
--- a/lang/python/python3-version.mk
+++ b/lang/python/python3-version.mk
@@ -13,7 +13,7 @@ PYTHON3_VERSION_MICRO:=9
 PYTHON3_VERSION:=$(PYTHON3_VERSION_MAJOR).$(PYTHON3_VERSION_MINOR)
 
 PYTHON3_SETUPTOOLS_PKG_RELEASE:=1
-PYTHON3_PIP_PKG_RELEASE:=1
+PYTHON3_PIP_PKG_RELEASE:=2
 
 PYTHON3_SETUPTOOLS_VERSION:=40.6.2
 PYTHON3_PIP_VERSION:=18.1
diff --git a/lang/python/python3/files/python3-package-pip.mk b/lang/python/python3/files/python3-package-pip.mk
index 0f8476c18..80fa6868c 100644
--- a/lang/python/python3/files/python3-package-pip.mk
+++ b/lang/python/python3/files/python3-package-pip.mk
@@ -12,7 +12,7 @@ $(call Package/python3/Default)
   DEPENDS:=+python3 +python3-setuptools +python-pip-conf
 endef
 
-define Package/python3-pip/install
+define Py3Package/python3-pip/install
 	$(INSTALL_DIR) $(1)/usr/bin $(1)/usr/lib/python$(PYTHON3_VERSION)/site-packages
 	$(CP) $(PKG_BUILD_DIR)/install-pip/bin/pip3* $(1)/usr/bin
 	$(CP) \
-- 
2.20.1

